packages/x-components/src/views/home/Home.vue

```diff
@@ -491,4 +533,7 @@
     height: calc(100% - 20px);
     margin: 10px;
   }
+  .x-next-query-preview__result {
+    max-width: 180px !important;
```

Comment:
inline the styles in the element, no need to add more complex CSS for this view:

`style="max-width: 180px"`